data/tlg0317/tlg002/tlg0317.tlg002.1st1K-grc1.xml

```diff
@@ -0,0 +1,537 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+ <teiHeader xml:lang="eng">
+  <fileDesc>
+   <titleStmt>
+    <title xml:lang="lat">Acta Joannis (recenscio)</title>
```

Comment:
No `<author>`?

Reply:
The author is the same as the title of the work, i.e., "Acta Joannis", so I don't think it is necessary.

Reply:
As with the previuos file, we should add the author here,too.